### PR TITLE
Add 1.8 LivingEntities

### PR DESCRIPTION
**Pull request by gabizou**
This adds Guardians, Rabbits, and Endermites to the API. The ID's are pulled from decompiled 1.8 source.

### CODE REVIEW

**Comment by turt2live** [comment]
Might want to take a second look at your java docs

**Comment by ZephireNZ** [comment]
Guardians are set as elders by a true/false value in their NBT, not by a type id. Get rid of the interger part of GuardianType?

File: src/main/java/org/bukkit/entity/Guardian.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package org.bukkit.entity;
+
+/**
+ * Represents a Guardian.
+ */
+public interface Guardian extends Monster, WaterMob {
+
+    /**
+     * Gets the current type of this skeleton.
```
Comment (turt2live) [inline src/main/java/org/bukkit/entity/Guardian.java]: Please fix your javadocs (x8)

**Comment by turt2live** [comment]
@gabizou Please include the method definitions for all missing entities, including those from #25. That would help the project progress further into the "it has all the things" portion of development :)

**Pull request closed without merging**